### ISSUE TITLE
UID

### ISSUE THREAD

**Issue by nipnipj** [glikely/obs-ptz#193]
Hello!
Can I use this software to watch from a IPCAM that uses UID. I can use my camera from a software named "camhi".
My device also provides info like IP adresss, subnet mask, gateway, DNS. Can I use this info within this plugin?

**Comment by glikely** [comment]
No, this plugin only provides PTZ control of cameras. It does not provide a vidoe stream
